### PR TITLE
improvement/CLDSRV-91 Bump Arsenal to 8.1.18

### PR DESCRIPTION
**Pull request by KillianG**
Bump arsenal to 8.1.18 to bring fix for ARTESCA-3109

### CODE REVIEW

**Comment by miniscruff** [comment]
You will need to run `yarn` to update the package lock as well

**Comment by ghost** [comment]
I think the ticket is CLDSRV-92, not CLDSRV-91 :slightly_smiling_face: 

Edit: 91 exisits, so I  updated the other ticket name to make it clearer

Edit 2: We shouldn't have any changes in the 8.3 integration branch, do you know why we have two new packages in the yarn lock file?

**Review by ghost — APPROVED** [review]

**Comment by KillianG** [comment]
> I think the ticket is [CLDSRV-92](https://scality.atlassian.net/browse/CLDSRV-92), not [CLDSRV-91](https://scality.atlassian.net/browse/CLDSRV-91) slightly_smiling_face
> 
> Edit: 91 exisits, so I updated the other ticket name to make it clearer

91 is the bump and 92 is the release

**Comment by KillianG** [comment]
> Edit 2: We shouldn't have any changes in the 8.3 integration branch, do you know why we have two new packages in the yarn lock file?

I don't know.. this arsenal version is the same that is already used in 8.3 there shouldnt be any changes in lock but there is..

**Comment by ghost** [comment]
Looking at the dev/8.2 branch, can we bump package.json version in this PR directly? The current version is 8.2.17 and we want to release in 8.2.18. You can refer to the release doc in dev/8.2, I think before we were releasing right after bumping the version in package.json.

**Review by welansari — APPROVED** [review]
LGTM

File: yarn.lock [hunk 1]
```diff
@@ -6774,7 +6774,7 @@ sprintf-js@~1.0.2:
   resolved "https://codeload.github.com/scality/sproxydclient/tar.gz/e45c547833f03b1fa0409e0499433dcb386442f5"
   dependencies:
     async "^3.1.0"
-    werelogs scality/werelogs#351a2a3
+    werelogs scality/werelogs#8.1.0
```
Comment (jbertran) [inline yarn.lock:6777]: Same concern here - we somehow went from that same commit to 8.1.0? But 8.1.0 is 2 years ahead time-wise

```
git show tags/8.1.0

tag 8.1.0
Tagger: Rahul Padigela <rahul.padigela@scality.com>
Date:   Fri Apr 9 17:02:24 2021 -0700

v8.1.0

commit e8f8287 (tag: 8.1.0)
Author: Rahul Padigela <rahul.padigela@scality.com>
Date:   Fri Apr 9 17:02:23 2021 -0700

    v8.1.0
```

File: yarn.lock [hunk 2]
```diff
@@ -1324,7 +1324,7 @@ bucketclient@scality/bucketclient#8.1.0:
   dependencies:
     agentkeepalive "^4.1.3"
     arsenal scality/Arsenal#8ed8478
-    werelogs scality/werelogs#8.1.0
+    werelogs scality/werelogs#351a2a3
```
Comment (jbertran) [inline yarn.lock]: I'm concerned here - we've pinned a commit that is *NOT* a tag. I'm not sure how that happened, it looks like we made a weird release.

```
git show 351a2a3

commit 351a2a3
Merge: 1a6e052 4e0d97c
Author: Katherine Laue <Katherine.laue@wustl.edu>
Date:   Fri Jun 28 14:03:48 2019 -0700

    Merge remote-tracking branch 'origin/w/7.5/improvement/S3C-2292-upgrade-nodejs' into w/8.0/improvement/S3C-2292-upgrade-nodejs
```

**Review by jbertran — CHANGES_REQUESTED** [review]
Can you investigate that weird version switch? I might be missing something in these dependencies.

**Review by jbertran — APPROVED** [review]

**Comment by KillianG** [comment]
werelogs problem was due to the waterflow because someone bumped sproxyd version in v7.4 and waterflow updated 8.2 as well with the old version of werelogs

**Pull request merged**